FashionConnect_ui/src/components/CreateCollection/CreateCollection.jsx

```diff
@@ -0,0 +1,37 @@
+import "./CreateCollection.css";
+import React ,{useState} from "react";
+import { useNavigate } from "react-router";
+
+export default function CreateCollection({userId}){
+    const [name, setName] = useState('');
+    const [description, setDescription] = useState('');
+    const navigate = useNavigate();
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+
+        // Make a POST request to create a new collection
+        const response = await fetch('http://localhost:3000/collections', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ name, description , userId}),
+        });
+
+        if (response.ok) {
+        setName('');
```

Comment:
nit: Don't forget to format your files! You can enable format on save. Let me know if you need help setting that up.